packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -85,9 +85,9 @@
 				></text-edit>
 
 				<code-node-editor
-					v-if="editorType === 'codeNodeEditor' && isCodeNode(node)"
+					v-if="editorType === 'codeNodeEditor' && isCodeNode(node) && editorContent"
```

Comment:
what's the purpose of the `&& editorContent` here?
if the value is empty, do we not want to render the editor ?

Reply:
Exactly, my goal was not to render component if this new logic fails to find a property but, since `value` is an optional prop for the editors, maybe we can skip this check...

Reply:
yeah. I removed the three instances of `&& editorContent`, and didn't see any issue.